cmd/gitops/root/cmd.go

```diff
@@ -85,6 +88,31 @@ func RootCmd() *cobra.Command {
 				fmt.Fprintf(os.Stderr, "Error: %v\n", err)
 				os.Exit(1)
 			}
+
+			_, err = config.GetConfig(false)
+			if err != nil {
+
+				prompt := promptui.Prompt{
+					Label:     "Creating config file...would you like to turn on analytics to help us improve our product",
```

Comment:
💡  Do we need a space after the ellipsis? And should we capitalize `w`? At least it will be read easier? For some reason, in other log messages we usually add a space before the ellipsis too, maybe it's a convention in CLI?